wireapp/wire-android-reloaded
# fix: crash when removing member from group conversation [AR-2038]

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

App was crashing with a NPE when trying to remove a member from a conversation because the `conversationId` was null as it was not being initialised on the `OtherUserProfileScreenViewModel.state`

### Testing

#### Test Coverage (Optional)


#### How to Test

Just navigate to a group where you are an admin, then try to remove a group member from his user profile screen and see that the confirmation dialog appears and that it removes the member from the group.

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
